contracts/FungibleToken.cdc

```diff
@@ -193,6 +195,24 @@ pub contract interface FungibleToken {
                     "New Vault balance must be the sum of the previous balance and the deposited Vault"
             }
         }
+
+        /// Function that returns all the Metadata Views implemented by a Fungible Token
+        ///
+        /// @return An array of Types defining the implemented views. This value will be used by
+        ///         developers to know which parameter to pass to the resolveView() method.
+        ///
+        pub fun getViews(): [Type]{
+            return []
+        }
+
+        /// Function that returns a Metadata View out of a Fungible Token
```

Comment:
```suggestion
        /// Function that resolves a metadata view for this fungible token by type.
```

Nitpick, but IMO I don't think we should treat "Metadata View" and "Fungible Token" as capitalized proper nouns in comments and documentation. I don't feel too strongly about this, though.

Reply:
I'm never sure with this details about use of English, so I'll agree with you!!